installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -81,7 +62,7 @@ metadata:
 data:
   "app-config-rhdh.yaml": |
     app:
-      title: Red Hat Developer Hub
+      title: {product}
```

Comment:
The `{product}` doesn't work in yaml block.
```suggestion
      title: Red Hat Developer Hub
```

Reply:
you should be able to have attribute expansion in code blocks. See https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/105/files#diff-1c0f0a6cd4adb348d1caf0302d7d7261c9e0b887b34e557ead9474bc496bcb83 where I've been trying to get ya'll to start using variables in code snippets for a month :D